microsoftgraph/microsoft-graph-explorer-v4
# Feature: upgrade fabric version

## Overview

Upgrades react fabric version to latest
Fixes breaking changes introduced

Fixes #579
Fixes #593 
Fixes #617 

### Demo

![image](https://user-images.githubusercontent.com/58787602/86906506-51922a00-c11c-11ea-838d-257303cf0eb6.png)


### Notes

Known issues:
Selecting an item and heading over to the next page loses the previous selection. 
This is however not expected as a heavy use-case since the search options are available and the modify permissions tab

## Testing Instructions

* Log in
* Select permissions
* Expand one section 
* Scroll to the bottom then back up again
* See whether the UI has blank spaces
